DemoApp/DemoApp/Views/Components/SectionHeader/SectionHeaderDemoView.swift

```diff
@@ -0,0 +1,27 @@
+import SwiftUI
+
+struct SectionHeaderDemoView: View {
+    var body: some View {
+        VStack(spacing: .spacingM) {
+            SectionHeader(title: "Section")
+
+            SectionHeader(title: "Section", actionView: SystemChevron.init, onClick: {})
+
+            SectionHeader(title: "Section", actionView: {
+                Picker("Section", selection: .constant("1")) {
+                    Text("1")
+                    Text("2")
+                    Text("3")
+                    Text("4")
+                }
+            })
+        }
+        .padding(.spacingM)
```

Comment:
in demo views is nice to add the `.navigationTitle` modifier so we can see at a glance which demo we are looking at